<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenProjectWindow</class>
 <widget class="QDialog" name="OpenProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打开项目</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>项目名:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>最近项目</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="open">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="cancel">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBox">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>全部日期</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最近一天</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最近一周</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最近一月</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最近半年</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最近一年</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
